_posts/minecraft/bedrock/2024-07-13-bedrock-rp.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title:  "Resource Packs"
+category: 
+    - Bedrock
+    - Getting-started
+tags: Server
+description: "Uploading Resource Packs into your Bedrock Server"
+permalink: /minecraft/bedrock/server/resource-pack
+author: "Kuroi Jigoku"
+toc: false
+---
+
+## Introduction
+
+Resource packs in Minecraft can significantly enhance the visual and auditory experience of the game. For server owners, offering custom resource packs can create a unique environment for players. This guide will walk you through the steps to upload and apply a resource pack to your Bedrock Minecraft server, ensuring all players can enjoy the customized experience you’ve designed. Whether you're adding custom textures, sounds, or user interfaces, follow these instructions to seamlessly integrate your resource pack with your Bedrock server.
+
+## Acquiring the resource packs
+
+There are several popular websites where you can find a variety of resource packs for Minecraft Bedrock Edition. Here are some of the most well-known ones:
+
+- [Planet Minecraft](https://www.planetminecraft.com/ "A popular platform for sharing Minecraft content, including resource packs, maps, and skins. The site has a dedicated section for Bedrock Edition.")
+- [MCPE DL](https://mcpedl.com/ "A website specifically focused on Minecraft Pocket Edition and Bedrock Edition, offering a wide range of resource packs, mods, maps, and more.")
+- [Minecraft Forum](https://www.minecraftforum.net/ "A long-standing community forum for Minecraft players, featuring a section dedicated to resource packs for all editions, including Bedrock.")
+- [Curseforge](https://www.curseforge.com/minecraft-bedrock "Known for hosting mods and addons for various games, CurseForge also has a section for Minecraft Bedrock Edition resource packs.")
+
+You can also upload your own custom resource pack as well.
+
+{: .warning}
+
+> **Compatibility Issues**: Ensure the resource pack is compatible with the version of Minecraft Bedrock Edition and any server plugins or mods installed. Incompatible packs may cause crashes or visual glitches.
+
+## Installation
+
+1. Download and extract the resource pack. If the file is in `.mcpack` format, rename the file extension to `.zip` format and extract it.
+![Renaming to zip](/assets/images/posts/rename.webp)
+
+2. In the extracted folder, there is a `manifest.json` file. Open it with Notepad or any other Text Editor.
+
+3. Minimize the file for now, we will need it later on.
+
+> In this tutorial, We will use SFTP for transferring files.
+> You may refer to [this tutorial](https://kb.falixnodes.net/falix/dashboard/general/sftp)
+
+## Transferring Files
+
+1. Log in to your Minecraft Server's SFTP.
```

Comment:
Perhaps using login is not the best idea, use connect
It also doesn't have to be a Minecraft server. Just say server.